src/cargo/ops/cargo_output_metadata.rs

```diff
@@ -39,10 +39,47 @@ pub fn output_metadata<'a>(opt: OutputMetadataOptions,
 #[derive(RustcEncodable)]
 pub struct ExportInfo {
     packages: Vec<Package>,
-    resolve: Resolve,
+    resolve: MetadataResolve,
     version: u32,
 }
 
+struct MetadataResolve(Resolve);
```

Comment:
Could you add a brief comment here explaining why this `Encodable` implementation is different than that of the lock file?